scarlet/renderer.py

```diff
@@ -194,18 +193,6 @@ def __init__(self, data_frame, model_frame, convolution_type="fft", padding=10):
         )
         self.diff_kernel = fft.match_psf(psf_fft, model_psf_fft, padding=padding)
 
-    def get_model(self, *parameters):
-        def transform(model):
-            # restrict to observed channels
-            model_ = self.map_channels(model)
-            # convolve observed channels
-            model_ = self.convolve(model_)
-            # adjust spatial shapes
-            model_ = match_shape(model_, self.data_frame, self.slices)
```

Comment:
This line preserved the output shape, i.e. if the data frame is larger/smaller than the model frame. Why is this not needed any longer?